src/PixiEditor/Views/Overlays/TransformOverlay/TransformUpdateHelper.cs

```diff
@@ -68,23 +68,15 @@ public static ShapeCorners? UpdateShapeFromCorner
             VecD leftNeighborPos = TransformHelper.GetAnchorPosition(corners, leftNeighbor);
             VecD rightNeighborPos = TransformHelper.GetAnchorPosition(corners, rightNeighbor);
 
-            double angle = corners.RectRotation;
-            if (double.IsNaN(angle))
-                angle = 0;
-
-            if (scaleFromCenter)
-            {
-                return ScaleCornersFromCenter(corners, targetCorner, desiredPos, angle);
-            }
-
-            // find positions of neighboring corners relative to the opposite corner, while also undoing the transform rotation
-            VecD targetTrans = (targetPos - oppositePos).Rotate(-angle);
-            VecD leftNeighTrans = (leftNeighborPos - oppositePos).Rotate(-angle);
-            VecD rightNeighTrans = (rightNeighborPos - oppositePos).Rotate(-angle);
+            // find positions of neighboring corners relative to the opposite corner
+            VecD targetTrans = targetPos - oppositePos;
+            VecD leftNeighTrans = leftNeighborPos - oppositePos;
+            VecD rightNeighTrans = rightNeighborPos - oppositePos;
```

Comment:
Most of the changes are just me removing un-rotation and re-rotation, I added comments to the 3 places that actually matter for scaling relative to center